enos/modules/iam_setup/main.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_caller_identity" "current" {}
+
+variable "test_id" {}
+
+locals {
+  user_email = split(":", data.aws_caller_identity.current.user_id)[1]
+}
+
+resource "aws_iam_user" "boundary" {
```

Comment:
Should we toss something in the README about this being restricted to certain accounts?